main.js

```diff
@@ -0,0 +1,101 @@
+
+//Show and Hide The news Sections
+var controlElement = document.querySelector('#show');
+controlElement.addEventListener('click', showHide);
+var content = document.querySelector('#mybutton');
+function showHide(event) {
+    event.preventDefault();
+    if (content.className === "hide") {
+        content.className = "";
+    }
+    else  {
+        content.className = "hide";
+    }
+}
+
+   
+
+
+//Form Selectors //
+
+var userEmail = document.querySelector('#useremail');
+var userName = document.querySelector('#username');
+var userPhone = document.querySelector('#userphone');
+
+var sbumitButton = document.querySelector('#submit-input');
+sbumitButton.addEventListener('click', submitForm);
+
+function submitForm(event) {
+ event.preventDefault();
+    verifyUserEmail();
+    verifyUserName();
+    verifyUserPhone();
+    if (verifyUserEmail() && verifyUserName() && verifyUserPhone()) {
```

Comment:
you are calling your validation functions twice without a reason. Once in line 30-33, then inside the if. Instead you can just put them in variables, i.e.

```javascript
var isEmailValid = verifyUserEmail();
var isUserNameValid = verifyUserName();

if (isEmailValid && isUserNameValid) {
 // alert('thanks');
}
```